### PR TITLE
Format markdown files with prettier

### PR DESCRIPTION
**Pull request by rictic**
Ran `npm run format:prettier`

### CODE REVIEW

File: packages/localize/CHANGELOG.md [hunk 1]
```diff
@@ -131,19 +131,11 @@ Before:
 class HomePage {
   hello() {
     // msgdesc: Greeting to Earth
-    return msg(
-      html`
-        Hello World
-      `
-    );
+    return msg(html` Hello World `);
```
[the remark this answers is not in the document]
Comment (rictic) [reply packages/localize/CHANGELOG.md:134]: Would you prefer I remove the leading and trailing spaces, or try to opt these code samples out of prettier's formatting?

File: packages/localize-tools/CHANGELOG.md [hunk 2]
```diff
@@ -159,19 +159,11 @@ Before:
 class HomePage {
   hello() {
     // msgdesc: Greeting to Earth
-    return msg(
-      html`
-        Hello World
-      `
-    );
+    return msg(html` Hello World `);
```
Comment (aomarks) [inline packages/localize-tools/CHANGELOG.md:162]:
```suggestion
    return msg(html`Hello World`);
```

File: packages/localize-tools/CHANGELOG.md [hunk 3]
```diff
@@ -159,19 +159,11 @@ Before:
...
   }
   goodbye() {
     // msgdesc: Farewell to Earth
-    return msg(
-      html`
-        Goodbye World
-      `
-    );
+    return msg(html` Goodbye World `);
```
Comment (aomarks) [inline packages/localize-tools/CHANGELOG.md:166]:
```suggestion
    return msg(html`Goodbye World`);
```

File: packages/localize-tools/CHANGELOG.md [hunk 4]
```diff
@@ -181,24 +173,14 @@ After:
...
   hello() {
-    return msg(
-      html`
-        Hello World
-      `,
-      {
-        desc: 'Home page / Greeting to Earth'
-      }
-    );
+    return msg(html` Hello World `, {
```
Comment (aomarks) [inline packages/localize-tools/CHANGELOG.md:176]:
```suggestion
    return msg(html`Hello World`, {
```

File: packages/localize-tools/CHANGELOG.md [hunk 5]
```diff
@@ -181,24 +173,14 @@ After:
...
   goodbye() {
-    return msg(
-      html`
-        Goodbye World
-      `,
-      {
-        desc: 'Home page / Farewell to Earth'
-      }
-    );
+    return msg(html` Goodbye World `, {
```
Comment (aomarks) [inline packages/localize-tools/CHANGELOG.md:181]:
```suggestion
    return msg(html`Goodbye World`, {
```

**Review by aomarks — CHANGES_REQUESTED** [review]
Found a few more :)

**Comment by rictic** [comment]
Applied!

**Review by aomarks — APPROVED** [review]

**Pull request merged**